akka-stream-tests/src/test/java/akka/stream/javadsl/SourceTest.java

```diff
@@ -626,6 +626,17 @@ public void mustRepeat() throws Exception {
     for (Integer i : result) assertEquals(i, (Integer) 42);
   }
 
+  @Test
+  public void mustRepeatForDocs() throws Exception {
+    // #repeat
+    Source<Integer, NotUsed> source = Source.repeat(42);
+    CompletionStage<List<Integer>> f = source.take(17).runWith(Sink.seq(), system);
+    // #repeat
+    final List<Integer> result = f.toCompletableFuture().get(3, TimeUnit.SECONDS);
+    assertEquals(result.size(), 17);
+    for (Integer i : result) assertEquals(i, (Integer) 42);
```

Comment:
Matter of taste but I like the non-testy ones with printlns and a comment saying what they'd print better than collecting into a seq. Not covered by regression test but more readable for the end user which is perhaps more important.

Reply:
I agree with that. It also removes the need to assign it to a `f`. 

Would it be more readable if we simply write:

```java
Source
	.repeat(43)
    .take(17)
    .runWith(Sink.println(), system)
```

ps: `Sink.println()`, I made this up. I don't know if such a sink exists

Reply:
I guess what I was looking for is something like `Sink.foreach(println)`